data-imports/clean_utils.py

```diff
@@ -139,7 +139,9 @@ def clean_bbl(table, boro, block, lot):
     return sql + '''
     UPDATE {table} SET {block} = REPLACE({block}, ' ', 0);
     UPDATE {table} SET {block} = REPLACE({block}, '`', '');
+    UPDATE {table} SET {block} = preg_replace('/[^0-9]/', '', {block});
     UPDATE {table} SET {lot} = REPLACE({lot}, ' ', 0);
+    UPDATE {table} SET {lot} = preg_replace('/[^0-9]/', '', {lot});
```

Comment:
Yea, it actually is super unclear as to whether the `'  '` -> `0` replacement is valid. 

My suspicion is the spaces are the result of bad scans or manual errors. A good place to start would be to find (from the initial import) some entries with spaces, and then compare the address against a different dataset. Or even use the [NYC BBL tool](http://webapps.nyc.gov:8084/CICS/fin1/find001I)

What would useful is finding if these addresses exhibit a pattern. As in, if all spaces are just that (empty space), then we could simply close them up and the regex is valid. However, if there are cases where spaces represent missing, unknown digits, we may want to mark these as having invalid data. We could then add an additional step to run these through an additional step that overwrites the BBL with data from the NYC BBL tool.

And yep, no particular rush on this...